src/viewProviders/RokuAutomationViewViewProvider.ts

```diff
@@ -9,6 +9,7 @@ import { ViewProviderId } from './ViewProviderId';
 import { ViewProviderCommand } from './ViewProviderCommand';
 import { ViewProviderEvent } from './ViewProviderEvent';
 import { WorkspaceStateKey } from './WorkspaceStateKey';
+import { readFileSync, writeFileSync } from 'fs';
```

Comment:
@TwitchBronBron any thought on importing this way?

Reply:
I don't have any fundamental issues with this pattern, but I think we import as `fs` everywhere else in the codebase so I'd say let's do that for consistency sake. 

Also, let's use `fs-extra` instead which comes with a few extra protections. 

```suggestion
import * as fsExtra from 'fs-extra';
```

Then usage, you can use `.outputFileSync` instead of `writeFileSync` so it auto-createes directories. 

https://github.com/jprichardson/node-fs-extra/blob/HEAD/docs/outputFile.md

Reply:
updated